experimental/Schemes/ProjectiveSchemes.jl

```diff
@@ -186,11 +186,11 @@ On ``X ⊂ ℙʳ(A)`` this returns ``r``.
 fiber_dimension(P::ProjectiveScheme) = P.r
 
 @Markdown.doc """
-    homogeneous_coordinate_ring(X::ProjectiveScheme)
+    homog_poly_ring(X::ProjectiveScheme)
```

Comment:
This does not look like an improvement. Can we have the old name or something without `homog`?

Reply:
(If it were internal and not exported I would not care)

Reply:
Ok, that should be no problem. But this should better be discussed on [this PR](https://github.com/oscar-system/Oscar.jl/pull/1242). Here, I would prefer to restrict to the discussion of how something like inheritance can be achieved in Oscar.